frontends/precinct-scanner/src/screens/setup_power_page.tsx

```diff
@@ -12,9 +12,9 @@ export function SetupPowerPage(): JSX.Element {
           <h1>
             No Power Detected <NoWrap>and Battery is Low</NoWrap>
           </h1>
-          <p>
-            Please ask a poll worker to plug-in the power cord for this machine.
-          </p>
+          <Text italic>
+            Please ask a poll worker to plug-in the power cord.
+          </Text>
```

Comment:
Should we just remove this screen since we are requiring power to be plugged in all the time whether or not battery is low?

Reply:
Yes. Can do. Seems like a separate PR though? I'll queue it up after this one.

Reply:
agreed!